homeassistant/components/sky_hub/manifest.json

```diff
@@ -2,5 +2,6 @@
   "domain": "sky_hub",
   "name": "Sky Hub",
   "documentation": "https://www.home-assistant.io/integrations/sky_hub",
+  "requirements": ["pyskyqhub==0.1.0"],
   "codeowners": []
```

Comment:
Do you want to become code owner?

Reply:
No. I started out to make a simple change to remove junk from my logs, and this has turned out to be more than I expected to take on.